packages/core/src/components/modal/modal.scss

```diff
@@ -0,0 +1,93 @@
+@import '~@atomium/scss-utils/index';
+
+.atom-modal {
+  &__close {
+    color: var(--color-neutral-light-1);
+    position: absolute;
+    right: var(--spacing-small);
+    top: var(--spacing-small);
+  }
+
+  &__close-icon {
+    color: var(--color-neutral-regular);
+    font-size: var(--spacing-large);
+  }
+
+  &__content {
+    color: var(--color-neutral-regular);
+    font: var(--text-body-medium);
+    height: 100%;
+    letter-spacing: var(--text-body-medium-letter);
+    padding: var(--spacing-large);
+    padding-top: 0;
+
+    &--divided {
```

Comment:
suggestion: this is a good scenario 

```
.content { 
  &--divided { 
```

Reply:
I undarstand but I think it's ok here. I'm respecting Block(atom-moda), Element(content), modifyer(divided). I like your tipe to avoid nested elements.